admin/tests/acceptance/authenticated/organizations/target-profiles-management_test.js

```diff
@@ -19,23 +19,23 @@ module('Acceptance | Organizations | Target profiles management', function (hook
     this.server.create('target-profile', { name: 'Profil cible du ghetto', ownerOrganizationId });
 
     // when
-    await visit(`/organizations/${ownerOrganizationId}/target-profiles`);
+    const screen = await visit(`/organizations/${ownerOrganizationId}/target-profiles`);
 
     // then
-    assert.dom('[aria-label="Profil cible"]').includesText('Profil cible du ghetto');
+    assert.dom(screen.getByLabelText('Profil cible')).includesText('Profil cible du ghetto');
   });
 
   test('should add a target profile to an organization', async function (assert) {
     // given
     const organization = this.server.create('organization');
 
     // when
-    await visit(`/organizations/${organization.id}/target-profiles`);
-    await fillIn('[aria-label="ID du ou des profil(s) cible(s)"]', '66');
-    await click('[aria-label="Rattacher un ou plusieurs profil(s) cible(s)"] button');
+    const screen = await visit(`/organizations/${organization.id}/target-profiles`);
+    await fillByLabel('ID du ou des profil(s) cible(s)', '66');
+    await clickByName('Valider');
```

Comment:
C'est bizarre ce libellé qui change. Est-ce parce que `clickByName` ne sait pas cibler avec le `aria-label` ? Si oui, ça me semble une bonne raison d'utiliser directement la librairie :

```suggestion
    await fillByLabel(screen.getByRole('textbox', { name: 'ID du ou des profil(s) cible(s)' }), '66');
    await click(screen.getByRole('button', { name: 'Rattacher un ou plusieurs profil(s) cible(s)' });
```

Reply:
en fait je pense qu'on ne peut pas avoir un aria label sur un bouton qui a du texte.. il va probablement avoir un probleme d'accessibilite avec 2 label pour un element

Reply:
Attention, ici le `Rattacher un ou plusieurs profil(s) cible(s)` était le aria-label ajouté sur l'élément HTML form. Pas le bouton.

D'un point de vue accessibilité on a pas besoin de fournir d'aria-label au form, le h2, et les champs suffisent s'ils sont bien nommé.
Donc j'ai retiré coté hbs cet aria-label inutile et le clickByName cible le texte dans le bouton.